ramp-frontend/ramp_frontend/utils.py

```diff
@@ -3,14 +3,36 @@
 """
 
 import logging
+from concurrent.futures import ThreadPoolExecutor
 
+from flask import (
+    copy_current_request_context,
+    current_app,
+)
 from flask_mail import Message
 
 from ramp_frontend import mail
 
 logger = logging.getLogger("RAMP-FRONTEND")
 
 
+def ensure_threadpoolexecutor_is_running(pool):
```

Comment:
@tomMoral could you provide some feedback here because I thought it was what you meant but I never really used the pool executor before.

Reply:
This seems fine like this.